.gitignore

```diff
@@ -15,3 +15,5 @@
 /log/*
 !/log/.keep
 /tmp
+
+.DS_Store
```

Comment:
@jeffminnear adding to or setting up some global gitignores the `.DS_Store` so its ignored by default in all your local git repos:

```bash
# global gitignores, create this file at ~/.gitignore_global and
# ensure the following git config is set:
# $> git config --global core.excludesfile ~/.gitignore_global

.DS_Store
```